packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkTest.kt

```diff
@@ -1263,6 +1281,40 @@ internal class DdSdkTest {
         }
     }
 
+    @Test
+    fun `𝕄 send long tasks 𝕎 frame time is over threshold() {}`(
+        @LongForgery timestampNs: Long,
+        @LongForgery(ONE_HUNDRED_MILLISSECOND_NS, ONE_SECOND_NS) frameDurationNs: Long,
+        @Forgery configuration: DdSdkConfiguration
+    ) {
+        // Given
+        val bridgeConfiguration = configuration.copy(
+            vitalsUpdateFrequency = "AVERAGE"
+        )
+
+        // When
+        testedBridgeSdk.initialize(bridgeConfiguration.toReadableJavaOnlyMap(), mockPromise)
+
+        // Then
+        argumentCaptor<Choreographer.FrameCallback> {
+            verify(mockChoreographer).postFrameCallback(capture())
+
+            // When
+            firstValue.doFrame(timestampNs)
+            firstValue.doFrame(timestampNs + frameDurationNs)
+
+            // then
+            verify(mockRumMonitor._getInternal()!!).updatePerformanceMetric(
+                RumPerformanceMetric.JS_FRAME_TIME,
+                frameDurationNs.toDouble()
+            )
+            verify(mockRumMonitor._getInternal()!!).addLongTask(
```

Comment:
I believe there is a very small chance of test flakiness here, when `timestampNs` is exactly `ONE_HUNDRED_MILLISSECOND_NS`, in this case `addLongTask` won't be called, because condition `if (it > longTaskThresholdNs)` will be `false`.